<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNeuronDialog</class>
 <widget class="QDialog" name="addNeuronDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Neuron</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>21</number>
   </property>
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item>
    <layout class="QGridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="0" column="1">
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>91</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="TextLabel1">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Device</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3" rowspan="2">
      <widget class="QListWidget" name="devListBox" native="true">
       <property name="currentItem" stdset="0">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QGroupBox" name="groupBoxLayer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioLayerInput">
          <property name="text">
           <string>I&amp;nput</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioLayerHidden">
          <property name="text">
           <string>Hidden</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioLayerOutput">
          <property name="text">
           <string>Output</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>34</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="addNeuronLabel">
       <property name="text">
        <string>Add neuron in position number:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SpinBox1">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>32</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>101</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAddNeuronOK">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAddNeuronCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 
 <customwidgets/>
 <tabstops>
  <tabstop>radioLayerHidden</tabstop>
  <tabstop>devListBox</tabstop>
  <tabstop>SpinBox1</tabstop>
  <tabstop>buttonAddNeuronOK</tabstop>
  <tabstop>buttonAddNeuronCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
